README.md

```diff
@@ -194,6 +194,7 @@ You can use keyboard shortcuts to navigate and perform certain actions:
  - `x` - mark/unmark current notification
  - `y` - archive current/marked notification(s)
  - `m` - mute current/marked notification(s)
+ - `d` - mark current/marked notification(s) as read here and on GitHub
```

Comment:
I arbitrarily chose `d` because the obvious choices of `r` for read and `m` for 'mark read' are already taken